server/src/main/java/com/decathlon/ara/cartography/AraExporter.java

```diff
@@ -36,7 +37,7 @@ public String getFormat() {
     }
 
     @Override
-    public byte[] generate(List<FunctionalityDTO> functionalities) {
+    public byte[] generate(List<FunctionalityDTO> functionalities, Map<String, String> requiredInfos) {
```

Comment:
A documentation could help

Reply:
The documentation of the new param is made in the parent class (`Exporter`),  and with the `@Override` annotation, this method inherit the doc automagically 😄